src/extensions/extension-manager.ts

```diff
@@ -45,15 +47,24 @@ export class ExtensionManager {
     return __non_webpack_require__.resolve('npm/bin/npm-cli')
   }
 
+  get packageJsonPath() {
+    return path.join(this.extensionPackagesRoot, "package.json")
+  }
+
   async load() {
     logger.info("[EXTENSION-MANAGER] loading extensions from " + this.extensionPackagesRoot)
+    if (fs.existsSync(path.join(this.extensionPackagesRoot, "package-lock.json"))) {
+      await fs.remove(path.join(this.extensionPackagesRoot, "package-lock.json"))
```

Comment:
Why do we need this with `--no-package-lock`?

Reply:
Current version has generated that file and it needs to be removed.. We can probably remove this cleanup at some point.